depends/cargo-checksum.sh

```diff
@@ -0,0 +1,13 @@
+echo "{\"files\":{$(
+find . -type f |  	# Get list of file paths
+grep -v $1 |		# Exclude Makefile hashes
+grep -v .stamp_ |	# Exclude Makefile stamps
+sed 's|^./||' |		# Remove leading ./
```

Comment:
Spotted a bug post-merge: this is supposed to match only leading "./", but actually matches any leading character followed by "/" as the second character.

```
$ echo 'X/foo' |sed 's|^./||'  # incorrect
foo
$ echo 'X/foo' |sed 's|^[.]/||'  # correct
X/foo
$ echo './foo' |sed 's|^[.]/||'  # correct
foo
```

Reply:
Similarly, the previous line should be ``grep -v '[.]stamp_' |``.

Reply:
Opened #3131.